source/blogs/posts/2017-02-25--wtf-is-sculpin.md

```diff
@@ -0,0 +1,206 @@
+---
+title: WTF is ... Sculpin
+date: 2017-02-25
+draft: true
+tags:
+    - wtf is
+    - sculpin
+featured_image: 
+    image: jackson.png
+    alt: Meet Jackson, the Sculpin mascot 
+---
+
+Sculpin is a static site generator written in PHP. And WTF is a static site genertor? It is a great tool for content that doesn’t need to be updated by multiple users on a regular basis, authentication, or any logic that requires server side communications. It can remove much of the complexity that comes with a CMS, as well as the need for server side technologies like PHP or a database server. Static site generator is great for my personal blog or the local astronomy group site, but not ideal from a large news organization that has much more dynamic content. Sculpin has the advantage of being written in a language I’m already familiar with; PHP. Some of the standout features of Sculpin include:
+
+* Writing content in markdown
+* TWIG templating
+* Syntax highlighting
+* Extremely fast
+* Secure (everything is client side)
+* Easily deployable (Can host static sites from github for free)
+
+When designing my blog I was looking for the path of least resistance for sharing some code snippets and jotting down a few of my opinions to share with the Interwebs. For someone that is already familiar with PHP Sculpin is the perfect tool to accomplish exactly this. How does one Sculpin?
+``
+## Installing Sculpin
+
+1. Download the starter kit
+
+```
+git clone https://github.com/sculpin/sculpin-blog-skeleton.git myblog
+cd myblog
+composer install
+```
+
+The starter kit is a skeleton build of Sculpin and is a great starting point for launching Sculpin as a blog.
+
+2. Run Sculpin
+
+```
+vendor/sculpin/sculpin/bin/sculpin generate --watch --server
+```
+
+Navigate to http://localhost:8000. BAM! You now have a boiler plate Sculpin site.
+
+## What's Happening?
+
+Magic! Okay maybe not magic. Sculpin comes with an executable bin, that we're running to parse all of the Twig templates, html, and markdown sources it can find to generate static html pages.
+
+The `--watch` simply tells Sculpin to watch for any file changes in order to rewrite any necessary HTML files. The `--server` launches PHP's built in web server which allows you to see your work in progress. The fact I don't need a working web server to get started makes it even easier to start creating content.
```

Comment:
> The `--watch` simply tells Sculpin to watch for ~~any~~ file changes in order to rewrite any necessary HTML files. The `--server` launches PHP's built in web server which allows you to see your work in progress. The fact **thatI** don't need a working web server to get started makes it even easier to start creating content.

too many instances of "any" in first sentence, suggest adding "that" in last sentence. Or rewording the sentence, because with the the "that" it feels awkward.